remoteit/desktop
# DESK-959 sessions refactor

### Description


Move session information out of device query.

### Test plan



1. Sign in
2. Go to a list with devices not loaded in the first 50
3. Connect from another app to one of the unloaded devices
4. You should see the connection notification and the status on the connections tab

### Ticket



<https://remoteit.atlassian.net/browse/DESK-959>
